Form/Type/ChangePasswordFormType.php

```diff
@@ -36,7 +36,7 @@ public function buildForm(FormBuilderInterface $builder, array $options)
             'label' => 'form.current_password',
             'translation_domain' => 'FOSUserBundle',
             'mapped' => false,
-            'constraints' => new UserPassword(),
+            'constraints' => new UserPassword(array('groups' => ['Default', 'ChangePassword'])),
```

Comment:
You hardcoded the value which is not good, this should be `$options['validation_groups']` instead and please use the old array syntax.